.appveyor.yml

```diff
@@ -56,6 +57,10 @@ install:
   - ps: Start-FileDownload "https://dl.google.com/android/repository/sdk-tools-windows-$env:ANDROID_SDK_VERSION.zip"
   - 7z x sdk-tools-windows-%ANDROID_SDK_VERSION%.zip -o%ANDROID_HOME% > nul
   - yes | %ANDROID_HOME%\tools\bin\sdkmanager.bat platform-tools
+  # Install rust
```

Comment:
There are Rust packages for Chocolatey (for both MSVC or GNU ABI), can we use these instead? See https://chocolatey.org/packages?q=rust.

Reply:
It might work however I did not see a single usage of `choco` in you appveyor script. I used the snippet from the official rustup app veyor configuration. Should I still switch to `choco`?

Reply:
See the calls to `cinst`, which is the short form of `choco install` :wink: If that works I'd prefer that to reduce the clutter a bit.